File: Frontend/SportMatch2/app/src/main/java/com/example/sportmatch/data/dto/CreateMatchDto.kt
package com.example.sportmatch.data.dto

import com.google.gson.annotations.SerializedName

data class CreateMatchDto(
    @SerializedName("hostId") val hostId: Int,
    @SerializedName("sportType") val sportType: String,
    @SerializedName("requestType") val requestType: String,
    @SerializedName("missingPlayers") val missingPlayers: Int,
    @SerializedName("description") val description: String?,
    @SerializedName("startTime") val startTime: String,
    @SerializedName("endTime") val endTime: String,
    @SerializedName("latitude") val latitude: Double,
    @SerializedName("longitude") val longitude: Double
)

File: Backend/SportMatchAPI/SportMatchAPI/DTO/CreateMatchDto.cs
namespace SportMatchAPI.DTO
{
    public class CreateMatchDto
    {
        public int HostId { get; set; }
        public string SportType { get; set; } = string.Empty;
        public string RequestType { get; set; } = string.Empty;
        public int MissingPlayers { get; set; }
        public string? Description { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
    }
}
